README.md

```diff
@@ -40,19 +40,20 @@ sudo podman run \
     --security-opt label=type:unconfined_t \
     -v $(pwd)/config.json:/config.json \
     -v $(pwd)/output:/output \
+    -v /var/lib/containers/storage:/var/lib/containers/storage \
     quay.io/centos-bootc/bootc-image-builder:latest \
     --type qcow2 \
     --config /config.json \
     quay.io/centos-bootc/fedora-bootc:eln
 ```
 
-### Using local containers
+NOTE: local storage is being used by default. If the `--local` flag is not provided, as in the above example,
+the latest image will be pulled into the local storage.
```

Comment:
There's no pulling involved here, right?  We're just...using the image that is local.  But now I'm confused as if it's default, why do we need to talk about it?

Reply:
We changed this up in https://github.com/osbuild/bootc-image-builder/pull/262 so that we always pull the container image

Reply:
This is a bad default, we are experiencing bad behaviour with ai-lab-recipes, because the images are huge.  Default should be to only pull if the image is missing.